src/lib/markbind/src/parser.js

```diff
@@ -566,7 +677,7 @@ Parser.prototype.includeFile = function (file, config) {
         try {
           processed = this._preprocess(d, context, config);
         } catch (err) {
-          err.message += `\nError while preprocessing '${file}'`;
+          err.message += `${err.stack}\nError while preprocessing '${file}'`;
```

Comment:
`err.stack` is not standard and should not be used in production code. 

[Reference here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack)